src/tools/fuzzing.h

```diff
@@ -1333,6 +1339,42 @@ class TranslateToFuzzReader {
     return builder.makeGlobalSet(pick(globals), value);
   }
 
+  Expression* makeTupleMake(Type type) {
+    assert(type.isMulti());
+    std::vector<Expression*> elements;
+    for (auto t : type.expand()) {
+      elements.push_back(make(t));
+    }
+    return builder.makeTupleMake(std::move(elements));
+  }
+
+  Expression* makeTupleExtract(Type type) {
+    assert(type.isSingle() && type.isConcrete());
+    Type tuple = getTupleType();
```

Comment:
Nit: How about `tupleType`? I think this can be confused as a tuple expression, which is to be created later.